js/krakenfutures.js

```diff
@@ -300,6 +300,8 @@ module.exports = class krakenfutures extends Exchange {
             // swap == perpetual
             let settle = undefined;
             let settleId = undefined;
+            const amountPrecision = this.parseNumber (this.parsePrecision (this.safeString (market, 'contractValueTradePrecision', '0')));
```

Comment:
@youknowone thanks, you're right for some market pairs (btc/usd:usd for instance) the amount precision is not 1, will merge it soon.